tests/integ/modin/io/test_read_snowflake_query_cte.py

```diff
@@ -165,7 +164,6 @@ def filterByRole(session: Session, tableName: String, role: String): DataFrame =
     assert_snowpark_pandas_equals_to_pandas_without_dtypecheck(snow_df, native_df)
 
 
-@pytest.mark.modin_sp_precommit
```

Comment:
can we add a todo to remind us to reenable it?

Reply:
Yes, added

Reply:
Can we track with a ticket?

Reply:
Yes I added [SNOW-1760467](https://snowflakecomputing.atlassian.net/browse/SNOW-1760467) to track this